docs/src/content/hardhat-network/docs/reference/index.md

```diff
@@ -61,7 +62,7 @@ The block gas limit to use in Hardhat Network's blockchain. Default value: `30_0
 
 #### `hardfork`
 
-This setting changes how Hardhat Network works, to mimic Ethereum's mainnet at a given hardfork. It must be one of `"byzantium"`, `"constantinople"`, `"petersburg"`, `"istanbul"`, `"muirGlacier"`, `"berlin"`, `"london"` and `"arrowGlacier"`. Default value: `"arrowGlacier"`
+This setting changes how Hardhat Network works, to mimic Ethereum's mainnet at a given hardfork. It must be one of `"byzantium"`, `"constantinople"`, `"petersburg"`, `"istanbul"`, `"muirGlacier"`, `"berlin"`, `"london"`, `"arrowGlacier"` and `"merge"`. Default value: `"arrowGlacier"`
```

Comment:
Aren't we missing one here? The latest HF?

Also, the new default should be `merge`.

Reply:
I added support for gray glacier in other PR: https://github.com/NomicFoundation/hardhat/pull/3087/commits/814a8e107dbeed6e027f5e1402fbc81af366924d

> Also, the new default should be merge.

I thought we were going to change the default after the mainnet merge, didn't we?

Reply:
The default will be changed after the merge.